src/cpu-template-helper/src/dump/host.rs

```diff
@@ -20,9 +29,73 @@ pub struct HostFingerprint {
 }
 
 pub fn dump() -> Result<HostFingerprint, Error> {
-    // TODO: Add implementation to dump host fingerprint.
+    let commands = vec![
+        "uname -r",
```

Comment:
Good point... would the minor version be enough? There are still things that can get through in patch releases though

Reply:
There certainly are things that may be significant in the patch releases. The point is they are likely to be visible via the guest configuration that we capture anyways. It should detect the majority of the changes.

Reply:
I have also been thinking of making the content of the fingerprint configurable (eg via options), but that may an overcomplication.

Reply:
yes we can. I get all info by running shell commands with a single function for simplicity, but using `libc` would allow hosts that don't have coreutils package. let me think.

Reply:
> Actually, the more I think about it, it looks like customers will be alarmed on every kernel patch update which could be a lot of false positives. Are we sure we need to capture uname?

I thought we wanted to make sure not to miss any changes, no matter how small.

Reply:
Regarding `libc::uname`, it seems we need some unsafe code like this:
```
    let mut buffer: libc::utsname = unsafe { std::mem::zeroed() };
    let ret = unsafe { libc::uname(&mut buffer) };
```
@pb8o Which one do you prefer? (I think we may assume coreutils package is installed on host, since it is a quite basic package including `cat`, `cp`, `ls`, `mv`, etc.)